src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -893,4 +893,8 @@ public void waitForVASTRequestWithAdUnit(NetworkTrafficInterceptor networkTraffi
     final String PATTERN = ".*output=xml_vast.*iu=" + encodedAdUnit + ".*";
     wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor, PATTERN);
   }
+
+  public void scrollBy(int x, int y) {
```

Comment:
That's another method called like that but in different object: [scrollBy()](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java#L44). Do you think we can re-use one of them instead of copying it?

Reply:
Unified! :)